tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>nit: This is why the detuning on q3 during this duration is constant equal to 0 <strong>and</strong> that q1 and q2 have the same local detuning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>